src/emulator/dataconnectEmulator.ts

```diff
@@ -59,7 +59,6 @@ export class DataConnectEmulator implements EmulatorInstance {
       auto_download: this.args.auto_download,
       listen: listenSpecsToString(this.args.listen),
       config_dir: this.args.configDir,
-      local_connection_string: this.getLocalConectionString(),
```

Comment:
Let's make sure to `/emulator/configure` with the local connection in a follow-up PR before the next CLI release.

Otherwise, `firebase emulators:start` would be broken.

@rosalyntan

Reply:
Is that covered in https://github.com/firebase/firebase-tools/pull/7266?